Update to 1.16, newer UI base images

palemtnrider opened the pull request:
Changes to generate and use apps/v1 instead of extensions/v1beta1.  Newer UI base images.

palemtnrider committed:
Update to 1.16, newer UI base images

D3nn reviewed (approved):
Looks good.  I sure hope you were able to do a global search and replace to get much of these!

palemtnrider committed:
General package updates

palemtnrider committed:
Update tslint rules based on new tslint package

palemtnrider committed:
Initial pass updating dependencies for the UI

palemtnrider committed:
NPM and inteceptor changes

Replaced subclass of httpclient with proper interceptor.
Updated version of angular

palemtnrider committed:
Initial work to remove old DnD.  Addl NPM cleanup

palemtnrider committed:
Simple Kustomize for local images

palemtnrider committed:
Angular 8+  target change for newer browsers

palemtnrider committed:
Fix init container drag and drop

palemtnrider committed:
Update to newer version of clarity

palemtnrider committed:
Update forms to clarity forms

palemtnrider committed:
Update select controls on new container dialogs
